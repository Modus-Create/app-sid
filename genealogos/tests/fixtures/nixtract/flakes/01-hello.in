{
	"flake_ref": "github:NixOS/nixpkgs?rev=ec045843b9981281bb8b676ed4af8e349fce820f",
	"attribute_path": "hello"
}
