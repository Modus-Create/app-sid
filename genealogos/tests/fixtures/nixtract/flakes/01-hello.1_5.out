{
  "bomFormat": "CycloneDX",
  "components": [
    {
      "bom-ref": "/nix/store/v5irq7wvkr7kih0hhnch5nnv2dcq8c4f-stdenv-linux",
      "description": "The default build environment for Unix packages in Nixpkgs",
      "externalReferences": [],
      "name": "stdenv-linux",
      "type": "application"
    },
    {
      "bom-ref": "/nix/store/ylriv849va759pz6sf9j06v9mwmh91w1-hello-2.12.1",
      "description": "",
      "externalReferences": [],
      "name": "hello",
      "purl": "pkg:generic/hello@2.12.1",
      "type": "application",
      "version": "2.12.1"
    },
    {
      "bom-ref": "/nix/store/h92a9jd0lhhniv2q417hpwszd4jhys7q-hello-2.12.1",
      "description": "A program that produces a familiar, friendly greeting",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://www.gnu.org/software/hello/manual/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "GPL-3.0-or-later"
          }
        }
      ],
      "name": "hello",
      "purl": "pkg:generic/hello@2.12.1",
      "type": "application",
      "version": "2.12.1"
    },
    {
      "bom-ref": "/nix/store/pa10z4ngm0g83kx9mssrqzz30s84vq7k-hello-2.12.1.tar.gz",
      "description": "",
      "externalReferences": [],
      "name": "hello",
      "type": "application"
    },
    {
      "bom-ref": "/nix/store/ypwhx9x3k8is4qmq1sgbzaggm5lcylhk-curl-8.4.0-dev",
      "description": "A command line tool for transferring files with URL syntax",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://curl.se/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "curl"
          }
        }
      ],
      "name": "curl",
      "purl": "pkg:generic/curl@8.4.0",
      "type": "application",
      "version": "8.4.0"
    },
    {
      "bom-ref": "/nix/store/1hjv10489fm00lckiicwsv3nlcm07njy-zlib-1.3-dev",
      "description": "Lossless data-compression library",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://zlib.net"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "Zlib"
          }
        }
      ],
      "name": "zlib",
      "purl": "pkg:generic/zlib@1.3",
      "type": "application",
      "version": "1.3"
    },
    {
      "bom-ref": "/nix/store/fpm6dfr2f288vgq3fmvfz9lmm7k6xlhq-nghttp2-1.57.0-dev",
      "description": "HTTP/2 C library and tools",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://nghttp2.org/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "MIT"
          }
        }
      ],
      "name": "nghttp2",
      "purl": "pkg:generic/nghttp2@1.57.0",
      "type": "application",
      "version": "1.57.0"
    },
    {
      "bom-ref": "/nix/store/sjm6zhcjq1l6ghcal55vmpy48zd0rhm7-perl-5.38.2",
      "description": "The standard implementation of the Perl 5 programming language",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://www.perl.org/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "Artistic-1.0"
          }
        }
      ],
      "name": "perl",
      "purl": "pkg:generic/perl@5.38.2",
      "type": "application",
      "version": "5.38.2"
    },
    {
      "bom-ref": "/nix/store/4jk7xwmpdb02cnv0w8jknprhxkbr8dyw-libssh2-1.11.0-dev",
      "description": "A client-side C library implementing the SSH2 protocol",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://www.libssh2.org"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "BSD-3-Clause"
          }
        },
        {
          "license": {
            "name": "libssh2 License"
          }
        }
      ],
      "name": "libssh2",
      "purl": "pkg:generic/libssh2@1.11.0",
      "type": "application",
      "version": "1.11.0"
    },
    {
      "bom-ref": "/nix/store/v6n306xw0v850dn01s9fdd0qjwh2j082-pkg-config-wrapper-0.29.2",
      "description": "A tool that allows packages to find out information about other packages (wrapper script)",
      "externalReferences": [
        {
          "type": "website",
          "url": "http://pkg-config.freedesktop.org/wiki/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "GPL-2.0-or-later"
          }
        }
      ],
      "name": "pkg-config-wrapper",
      "purl": "pkg:generic/pkg-config-wrapper@0.29.2",
      "type": "application",
      "version": "0.29.2"
    },
    {
      "bom-ref": "/nix/store/m8w6di2fqgblimidllyfc3fypgysrpgn-openssl-3.0.12-dev",
      "description": "A cryptographic library that implements the SSL and TLS protocols",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://www.openssl.org/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "Apache-2.0"
          }
        }
      ],
      "name": "openssl",
      "purl": "pkg:generic/openssl@3.0.12",
      "type": "application",
      "version": "3.0.12"
    },
    {
      "bom-ref": "/nix/store/ml1nkm5dvlmyp19vlbi4b2cxzky78qfb-libkrb5-1.21.2-dev",
      "description": "MIT Kerberos 5",
      "externalReferences": [
        {
          "type": "website",
          "url": "http://web.mit.edu/kerberos/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "MIT"
          }
        }
      ],
      "name": "libkrb5",
      "purl": "pkg:generic/libkrb5@1.21.2",
      "type": "application",
      "version": "1.21.2"
    },
    {
      "bom-ref": "/nix/store/3wvj2nbynari4iirc4yx3mhf26hbjr9s-libxcrypt-4.4.36",
      "description": "Extended crypt library for descrypt, md5crypt, bcrypt, and others",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://github.com/besser82/libxcrypt/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "LGPL-2.1-or-later"
          }
        }
      ],
      "name": "libxcrypt",
      "purl": "pkg:generic/libxcrypt@4.4.36",
      "type": "application",
      "version": "4.4.36"
    },
    {
      "bom-ref": "/nix/store/2kq5jql90fn5y2pd6c9vnjqffmf9ndjm-make-shell-wrapper-hook",
      "description": "",
      "externalReferences": [],
      "name": "make-shell-wrapper-hook",
      "type": "application"
    },
    {
      "bom-ref": "/nix/store/75nqgjskxzfzmq99rdl4isbpgkc1gjgf-keyutils-1.6.3-dev",
      "description": "Tools used to control the Linux kernel key management system",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://people.redhat.com/dhowells/keyutils/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "GPL-2.0-or-later"
          }
        }
      ],
      "name": "keyutils",
      "purl": "pkg:generic/keyutils@1.6.3",
      "type": "application",
      "version": "1.6.3"
    },
    {
      "bom-ref": "/nix/store/qy4h1wl3awid78vbbyvnn0div02kjv71-perl-5.38.2",
      "description": "The standard implementation of the Perl 5 programming language",
      "externalReferences": [
        {
          "type": "website",
          "url": "https://www.perl.org/"
        }
      ],
      "licenses": [
        {
          "license": {
            "id": "Artistic-1.0"
          }
        }
      ],
      "name": "perl",
      "purl": "pkg:generic/perl@5.38.2",
      "type": "application",
      "version": "5.38.2"
    },
    {
      "bom-ref": "/nix/store/26v33fxjpdn5wd06jqqjc8qbl3sc3sqp-die-hook",
      "description": "",
      "externalReferences": [],
      "name": "die-hook",
      "type": "application"
    }
  ],
  "dependencies": [
    {
      "dependsOn": [],
      "ref": "/nix/store/v5irq7wvkr7kih0hhnch5nnv2dcq8c4f-stdenv-linux"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/ylriv849va759pz6sf9j06v9mwmh91w1-hello-2.12.1"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/h92a9jd0lhhniv2q417hpwszd4jhys7q-hello-2.12.1"
    },
    {
      "dependsOn": [
        "/nix/store/ypwhx9x3k8is4qmq1sgbzaggm5lcylhk-curl-8.4.0-dev"
      ],
      "ref": "/nix/store/pa10z4ngm0g83kx9mssrqzz30s84vq7k-hello-2.12.1.tar.gz"
    },
    {
      "dependsOn": [
        "/nix/store/1hjv10489fm00lckiicwsv3nlcm07njy-zlib-1.3-dev",
        "/nix/store/4jk7xwmpdb02cnv0w8jknprhxkbr8dyw-libssh2-1.11.0-dev",
        "/nix/store/fpm6dfr2f288vgq3fmvfz9lmm7k6xlhq-nghttp2-1.57.0-dev",
        "/nix/store/m8w6di2fqgblimidllyfc3fypgysrpgn-openssl-3.0.12-dev",
        "/nix/store/ml1nkm5dvlmyp19vlbi4b2cxzky78qfb-libkrb5-1.21.2-dev",
        "/nix/store/sjm6zhcjq1l6ghcal55vmpy48zd0rhm7-perl-5.38.2",
        "/nix/store/v6n306xw0v850dn01s9fdd0qjwh2j082-pkg-config-wrapper-0.29.2"
      ],
      "ref": "/nix/store/ypwhx9x3k8is4qmq1sgbzaggm5lcylhk-curl-8.4.0-dev"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/1hjv10489fm00lckiicwsv3nlcm07njy-zlib-1.3-dev"
    },
    {
      "dependsOn": [
        "/nix/store/v6n306xw0v850dn01s9fdd0qjwh2j082-pkg-config-wrapper-0.29.2"
      ],
      "ref": "/nix/store/fpm6dfr2f288vgq3fmvfz9lmm7k6xlhq-nghttp2-1.57.0-dev"
    },
    {
      "dependsOn": [
        "/nix/store/3wvj2nbynari4iirc4yx3mhf26hbjr9s-libxcrypt-4.4.36"
      ],
      "ref": "/nix/store/sjm6zhcjq1l6ghcal55vmpy48zd0rhm7-perl-5.38.2"
    },
    {
      "dependsOn": [
        "/nix/store/1hjv10489fm00lckiicwsv3nlcm07njy-zlib-1.3-dev",
        "/nix/store/m8w6di2fqgblimidllyfc3fypgysrpgn-openssl-3.0.12-dev"
      ],
      "ref": "/nix/store/4jk7xwmpdb02cnv0w8jknprhxkbr8dyw-libssh2-1.11.0-dev"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/v6n306xw0v850dn01s9fdd0qjwh2j082-pkg-config-wrapper-0.29.2"
    },
    {
      "dependsOn": [
        "/nix/store/2kq5jql90fn5y2pd6c9vnjqffmf9ndjm-make-shell-wrapper-hook",
        "/nix/store/sjm6zhcjq1l6ghcal55vmpy48zd0rhm7-perl-5.38.2"
      ],
      "ref": "/nix/store/m8w6di2fqgblimidllyfc3fypgysrpgn-openssl-3.0.12-dev"
    },
    {
      "dependsOn": [
        "/nix/store/75nqgjskxzfzmq99rdl4isbpgkc1gjgf-keyutils-1.6.3-dev",
        "/nix/store/m8w6di2fqgblimidllyfc3fypgysrpgn-openssl-3.0.12-dev",
        "/nix/store/sjm6zhcjq1l6ghcal55vmpy48zd0rhm7-perl-5.38.2",
        "/nix/store/v6n306xw0v850dn01s9fdd0qjwh2j082-pkg-config-wrapper-0.29.2"
      ],
      "ref": "/nix/store/ml1nkm5dvlmyp19vlbi4b2cxzky78qfb-libkrb5-1.21.2-dev"
    },
    {
      "dependsOn": [
        "/nix/store/qy4h1wl3awid78vbbyvnn0div02kjv71-perl-5.38.2"
      ],
      "ref": "/nix/store/3wvj2nbynari4iirc4yx3mhf26hbjr9s-libxcrypt-4.4.36"
    },
    {
      "dependsOn": [
        "/nix/store/26v33fxjpdn5wd06jqqjc8qbl3sc3sqp-die-hook"
      ],
      "ref": "/nix/store/2kq5jql90fn5y2pd6c9vnjqffmf9ndjm-make-shell-wrapper-hook"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/75nqgjskxzfzmq99rdl4isbpgkc1gjgf-keyutils-1.6.3-dev"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/qy4h1wl3awid78vbbyvnn0div02kjv71-perl-5.38.2"
    },
    {
      "dependsOn": [],
      "ref": "/nix/store/26v33fxjpdn5wd06jqqjc8qbl3sc3sqp-die-hook"
    }
  ],
  "serialNumber": "urn:uuid:00000000-0000-0000-0000-000000000000",
  "specVersion": "1.5",
  "version": 1
}